WikipediaLibrary/TWLight
# Remove `/home` page

[//]: # (Thank you for uploading a PR to the Wikipedia Library!)

## Description
 - Removed the `/home` URL
 - Removed some CSS classes that were only used by the `/home` template
 - Deleted the `home.html` file
 - Deleted `HomepageView`

## Rationale
[//]: # (Why is this change required? What problem does it solve?)
With the new eligibility UX implemented, we no longer have a use for this intermediary page.

## Phabricator Ticket
[//]: # (Link to the Phabricator ticket)
[T285930](https://phabricator.wikimedia.org/T285930)

## How Has This Been Tested?
[//]: # (- Did you add tests to your changes? Did you modify tests to accommodate your changes?)
[//]: # (- Can this change be tested manually? How?)
1. Try navigating to `/home`. A 404 error should show
2. Log in as an ineligible account. The eligibility should show up on top of the `/users/my_library` page
3. Log in as an eligible account. The user can see the contents of `/users/my_library`

## Screenshots of your changes (if appropriate):
[//]: # (It can also be a GIF to prove that your changes are working)
![Screen Shot 2021-08-19 at 12 51 54](https://user-images.githubusercontent.com/7854953/130119375-7712bc0f-a1f8-4475-843f-8a3f361c0bc4.png)


## Types of changes
What types of changes does your code introduce? Add an `x` in all the boxes that apply:
